webrtc.html

```diff
@@ -1654,6 +1654,11 @@ <h4>Set the RTCSessionDescription</h4>
                               </li>
                             </ol>
                           </li>
+                          <li>
+                            <p>If the <a>media description</a> is rejected, and
+                            <var>transceiver</var> is not already stopped, <a>stop
+                            the RTCRtpTransceiver</a> <var>transceiver</var>.</p>
+                          </li>
```

Comment:
This is a normative change, and it appears in the wrong place I think, since it would seem to affect SLD(offer) here. Should it at least be inside the sub-steps directly above?

Reply:
Rejecting a media section and then leaving the transceiver unstopped is certainly the wrong thing to do in my opinion. I haven't tried to reformat this to see where it's actually indented to (it's pretty deep).